UI/SplashScreenSample/SplashScreenSample.Wasm/WasmScripts/AppManifest.js

```diff
@@ -1,6 +1,6 @@
 ﻿var UnoAppManifest = {
 
-    splashScreenImage: "Assets/SplashScreen.png",
+    splashScreenImage: "Assets/SplashScreen.scale-200.png",
```

Comment:
For web I think the 100x is probably the better option.

Reply:
I just followed our doc
https://platform.uno/docs/articles/splash-screen.html?tabs=tabid-vswin#step-5---webassembly